python_scripts/ExploreData.py

```diff
@@ -57,14 +57,15 @@ def HandleNanData(dataFrame, categoricalFeatureList):
 def HandleCategoricalData(dataFrame, categoricalFeatureList):
 	
 	for catfeat in categoricalFeatureList:
-		dataFrame = pd.concat([dataFrame, pd.get_dummies(dataFrame[catfeat],      sparse=True)])
+		print catfeat + " includes "+str(len(dataFrame[catfeat].unique()))+" different categorical values"
+		dataFrame = pd.concat([dataFrame, pd.get_dummies(dataFrame[catfeat],      sparse=False)],axis=1)
```

Comment:
I'm unfamiliar with xgboost so I'm going to have to assume that fitting categories to dummy binaries will work, but in a future version we may want to be more explicit in how we handle categorical data